packages/auth/src/app.ts

```diff
@@ -76,6 +77,7 @@ export interface AppServices {
   clientService: Promise<ClientService>
   grantService: Promise<GrantService>
   accessTokenRoutes: Promise<AccessTokenRoutes>
+  grantRoutes: Promise<GrantRoutes>
```

Comment:
This is so that `grantRoute` resolution below gives back proper type, not just `any`